### ISSUE TITLE
KeyError: 'eye_distance', in regress_kp(batch_list), celeba_wild.py.

### ISSUE THREAD

**Issue by LeonXu127** [xingzhehe/AutoLink-Self-supervised-Learning-of-Human-Skeletons-and-Object-Outlines-by-Linking-Keypoints#7]
In regress_kp(bctch_list) from celeba_wild.py, there are only 2 keys 'det_keypoints' and 'keypoints'. 
Should I add "'eye_distance': batch['eye_distance'].cpu()" to line 93 of model.py which is originally  "out_batch = {'keypoints': batch['keypoints'].cpu(), 'det_keypoints': kp.cpu()}"?
![QQ_1720710315921](https://github.com/xingzhehe/AutoLink-Self-supervised-Learning-of-Human-Skeletons-and-Object-Outlines-by-Linking-Keypoints/assets/54067417/a2d4db17-2ed3-4da4-b077-9e1916cc14ab)

**Comment by xingzhehe** [comment]
Yes. You will need `eye_distance` during inference. I think it can also be inferred from the GT keypoints, but I seemed to forget to modify the dataset-specific testing code.
